src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -159,11 +167,24 @@ export default function EnvironmentEditor({
         setEditModalOpen(true);
     };
 
+    const onTransitionEditClick = (sceneId) => {
+        const scene = scenes.find((scene) => scene.id === sceneId);
+
+        // TODO: get transition information (have to go up to scenario and match index from scene table)
+        setEditTransitionInfo(scene);
+        setEditTransitionModalOpen(true);
```

Comment:
If it's this much work, do you think a restructuring of our table for this is necessary? I don't really know, hence why I am asking